src/java/org/apache/cassandra/tools/nodetool/GetStreamThroughput.java

```diff
@@ -22,7 +22,7 @@
 import org.apache.cassandra.tools.NodeProbe;
 import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
 
-@Command(name = "getstreamthroughput", description = "Print the Mb/s throughput cap for streaming and entire SSTable streaming in the system")
+@Command(name = "getstreamthroughput", description = "Print the megabits per second throughput cap for streaming and entire SSTable streaming in the system")
```

Comment:
nit: Perhaps we should just say "...the throughput cap...", given there are two possible units?